pkg/metrics/server.go

```diff
@@ -142,6 +151,10 @@ func (s *Server) secureMetricsValid() bool {
 		log.Error(errors.Wrap(err, "secure metrics server is disabled"))
 		return false
 	}
+	if s.tlsConfigurer == nil {
+		utils.Should(errors.New("invalid TLS configurer"))
+		return false
```

Comment:
Probably should keep the same log statement that the secure metrics server is disabled here.

Reply:
Hmm I think this code doesn't even work, see my comment about the nil check on interface types.